sirepo/sim_data/madx.py

```diff
@@ -20,21 +20,39 @@ def fixup_old_data(cls, data):
                 m = dm['twissEllipseReport{}'.format(i)] = PKDict()
                 cls.update_model_defaults(m, 'twissEllipseReport')
                 m.dim = 'x' if i == 1 else 'y'
+        if 'bunchReport1' not in dm:
+            b_params = [('x', 'px'), ('y', 'py'), ('x', 'y'), ('t', 'pt')]
+            for (i, p) in enumerate(b_params):
+                m = dm['bunchReport{}'.format(i + 1)] = PKDict()
+                cls.update_model_defaults(m, 'bunchReport')
+                m.x = b_params[i][0]
+                m.y = b_params[i][1]
+        if 'initialTwissParams' not in dm:
+            for dim in ['x', 'y']:
+                m = dm['initialTwissParams'] = PKDict()
+                cls.update_model_defaults(m, 'initialTwissParams')
+                m.dim = dim
+        if 'particleTracking' not in dm:
+            m = dm['particleTracking'] = PKDict()
```

Comment:
I don't see the particleTracking model in the schema - was it replaced with something else and this line can be removed?

Reply:
Yes - there was already simulation.numberOfParticles so particleTracking model is no longer used.